quickstarts/microsoft.apimanagement/api-management-logs-to-moesif-using-eventhub-webapp/metadata.json

```diff
@@ -4,10 +4,9 @@
   "description": "The template will log API calls from Azure API Management to Moesif API analytics so you can understand customer API usage and resolve customer issues quickly.",
   "summary": "Powerful API analytics for Azure API Management to help you activate, understand, and monetize customers",
   "githubUsername": "Moesif",
-  "dateUpdated": "2022-06-08",
+  "dateUpdated": "2022-08-30",
   "type": "QuickStart",
   "environments": [
-    "AzureUSGovernment",
```

Comment:
Why are we removing USGov support?  (and not the other sovereign clouds?)

Reply:
Ok, I see the post above - that a transient problem though, not a statement of support.  If we deploy to usgovtexas does that work?

Reply:
Hi Thx for feedback. Somehow I missed your later comment, and based on first comment, I just pushed change to remove `AzureGermanCloud` and `AzureChinaCloud`. This quickstart is probably not getting used by any of sovereign clouds.

Yes, it does look like a `transient` problem, however, it has failed every run today (4 runs i think), and without this change, mutual customers are blocked from utilizing functionality. I am not sure how to use `usgovtexas`.

Request we merge this change, and then after the govcloud issues are resolved, we can easily re-add back the sovereign clouds.